recipes/ampligone/meta.yaml

```diff
@@ -24,12 +26,12 @@ requirements:
   run:
     - python >=3.8
     - pandas >=1.3
-    - pysam =0.19
+    - pysam #=0.19
```

Comment:
IMO these comments (`#=x.xx`) better be removed. It is a bit confusing now because they look like selectors

Reply:
```suggestion
    - pysam
```